src/Framework/Microsoft.AspNetCore.App.props

```diff
@@ -136,6 +139,7 @@
     <Dependency Include="System.Security.Permissions"                                  Version="$(SystemSecurityPermissionsPackageVersion)" />
     <Dependency Include="System.Text.Encodings.Web"                                    Version="$(SystemTextEncodingsWebPackageVersion)" />
     <Dependency Include="System.Threading.Channels"                                    Version="$(SystemThreadingChannelsPackageVersion)" />
+    <Dependency Include="System.Net.WebSockets.WebSocketProtocol"                      Version="$(SystemNetWebSocketsWebSocketProtocolPackageVersion)" />
```

Comment:
This appears to come from Microsoft.AspNetCore.Sockets:

```js
"Microsoft.AspNetCore.WebSockets/3.0.0-preview-t000": {
    "dependencies": {
      "Microsoft.AspNetCore.Http.Extensions": "3.0.0-preview-t000",
      "Microsoft.Extensions.Logging.Abstractions": "3.0.0-preview.19059.5",
      "Microsoft.Extensions.Options": "3.0.0-preview.19059.5",
      "System.Net.WebSockets.WebSocketProtocol": "4.6.0-preview.18619.1"
    },
    "runtime": {
      "lib/netcoreapp3.0/Microsoft.AspNetCore.WebSockets.dll": {
        "assemblyVersion": "3.0.0.0",
        "fileVersion": "3.0.0.19010"
      }
    }
  },
```

Reply:
It was removed recently https://github.com/aspnet/AspNetCore/pull/6646/files